docs/docs/mutations.md

```diff
@@ -0,0 +1,433 @@
+---
+title: Mutations
+---
+
+```typescript
+import { httpMutation } from '@angular-architects/ngrx-toolkit';
+```
+
+```typescript
+import { rxMutation } from '@angular-architects/ngrx-toolkit';
+```
+
+```typescript
+import { withMutations } from '@angular-architects/ngrx-toolkit';
+```
+
+```typescript
+// Optional, `concatOp` is the default.
+import { concatOp, exhaustOp, mergeOp, switchOp } from '@angular-architects/ngrx-toolkit';
+```
+
+## Basic Usage
+
+The mutations feature (`withMutations`) and methods (`httpMutation` and `rxMutation`) seek to offer an appropriate equivalent to signal resources for sending data back to the backend. The methods can be used in `withMutations()` or on their own.
+
+This guide covers
+
+- Why we do not use [`withResource`](./with-resource), and the direction on mutations from the community
+- Key Features ([summary](#key-features-summary) and [in depth](#key-features-in-depth)):
+  <!-- TODO (discuss): I think it is important to know it is `HttpClient` under the hood for stuff like interceptors and global stuff -->
```

Comment:
While it may be more of an implementation detail, it is important that people know global functions like interceptors will still work. While we can't hook into non-rjxs HttpClient API like Angular's source can do with `httpResource` and have to do this, I believe this is an important implementation detail to know about.